playbooks/robusta_playbooks/node_enrichments.py

```diff
@@ -135,10 +136,14 @@ def node_graph_enricher(node_event: NodeEvent, params: ResourceGraphEnricherPara
     """
     Get a graph of a specific resource for this node.
     """
-    start_at = datetime.now()
-    labels = {'node': node_event.get_node().metadata.name}
-
     node = node_event.get_node()
+    graph_enrichment = create_node_graph_enrichment(params, node)
+    node_event.add_enrichment([graph_enrichment])
+
+
+def create_node_graph_enrichment(params: ResourceGraphEnricherParams, node: Node):
```

Comment:
Maybe it would be better to add it in prometheus_enrichment_utils, like create_graph_enrichment

Reply:
I get it since `pod_enrichments` uses it but personally since this is node specific and this is just a small wrapper function I feel like its makes more sense location wise in the `node_enrichments`.